config.js

```diff
@@ -111,6 +111,8 @@ const JOB_QUEUE_TTL_SECONDS_SYNC = pe.TTL_KUE_JOBS_SYNC
 // set time interval for enableQueueStatsActivityLogs
 const queueStatsActivityLogsInterval = 60000;
 
+const DEFAULT_LIMIT = pe.DEFAULT_LIMIT || 10000;
```

Comment:
Also, maybe `const GET_REQUEST_DEFAULT_LIMIT = +pe.GET_REQUEST_DEFAULT_LIMIT || 10000;` (the plus sign to cast the env var to number, and if it can't be cast to number, it'll go to the "|| 10000")